rs/execution_environment/tests/execution_test.rs

```diff
@@ -1424,92 +1384,6 @@ fn canister_with_memory_allocation_cannot_grow_wasm_memory_above_allocation_wasm
     assert_eq!(err.code(), ErrorCode::CanisterOutOfMemory);
 }
 
-#[test]
-fn max_canister_memory_size_is_different_between_wasm32_vs_wasm64() {
```

Comment:
wasm memory limits are not effective when enforced via maximum canister memory size which is >500G in practice => there should be tests for wasm memory limits in the two modes (32-bit and 64-bit) independently of maximum canister memory size